tests/h/services/group_test.py

```diff
@@ -175,6 +184,26 @@ def test_created_by_excludes_other_groups(self, svc, db_session, factories):
     def test_created_by_returns_empty_list_for_missing_user(self, svc):
         assert svc.groupids_created_by(None) == []
 
+    @pytest.fixture
+    def with_groups(self, factories, user):
+        factories.OpenGroup(pubid="NOISE")
+        factories.OpenGroup(pubid="pubid1")
+        factories.Group(pubid="pubid2").members.append(user)
+
+        factories.OpenGroup(
+            pubid="authority_1a", authority="auth_1", authority_provided_id="auth_id_1a"
+        )
+        factories.Group(
+            pubid="authority_1b", authority="auth_1", authority_provided_id="auth_id_1b"
+        ).members.append(user)
```

Comment:
It's important to have a mixture of group types here, so we can check the user is being shown private ones they are a member of.